quiche/src/lib.rs

```diff
@@ -1888,7 +1909,8 @@ impl Connection {
             next_pkt_num: 0,
 
             peer_transport_params: TransportParams::default(),
-
+            peer_transport_params_track_unknown: config
```

Comment:
```suggestion

            peer_transport_params_track_unknown: config
```

Reply:
Thank you! I _think_ that I properly incorporated all these suggestions! I will close them just to save you the extra clicking, but if I missed anything, _please_ let me know!